packit_service/models.py

```diff
@@ -356,6 +353,38 @@ def __str__(self):
         return self.__repr__()
 
 
+class TaskResult(Task):
```

Comment:
we should inherit from Base actually and define the fields ourselves

I recall that @jpopelka did not like this in redis that we would store the data twice, but on the other hand: one copy is handled by celery and the other one by us to use - alternatively we could write code which would retrieve data from the celery tables, not sure if we want that

my preferred solution is to have two copies and full control of the data, rather then rely on celery

Reply:
Sorry guys, I have no idea why we need this and have no connection to even read the docs so feel free to do whatever you like. If it's just about having some redundant data, well, ok, disk space is cheaper than memory (redis), so, go ahead.

Reply:
Oki, we can fight about this on the next architecture meeting :))